DOCS/Users_Guide/Tutorials/CMAQ_UG_tutorial_WRF-CMAQ_build_gcc.md

```diff
@@ -231,6 +243,18 @@ Change directories to the CMAQ_HOME directory
 setenv NETCDF netcdf_root_gcc # Note please combine netCDF-C & Fortran Libraries (e.g. /usr/local/netcdf-4.7.0)
 setenv IOAPI  ioapi_root_gcc  (e.g. /usr/local/ioapi-3.2)
 setenv WRF_ARCH 34              # [1-75]  64 Bit Linux_x86 Compiler/Architecture options
+
+ #> I/O API, netCDF, and MPI include and library locations
+ setenv IOAPI_INCL_DIR   $IOAPI/ioapi/fixed_src   #> I/O API include header files
+ setenv IOAPI_LIB_DIR    $IOAPI/Linux2_x86_64gfort_openmpi_4.0.1_gcc_9.1.0   #> I/O API libraries
+ setenv NETCDF_LIB_DIR   $NETCDF/lib  #> netCDF C directory path
+ setenv NETCDF_INCL_DIR  $NETCDF/include  #> netCDF C directory path
+ setenv NETCDFF_LIB_DIR  $NETCDF/lib #> netCDF Fortran directory path
```

Comment:
@lizadams, I changed the config_cmaq.csh to include your changes, but I didn't change the NETCDFF_LIB_DIR and NETCDF_INCL_DIR to have the root $NETCDF, in the case that the left it as default and did not combine the NETCDF and NETCDFF directories. 


Additionally  in the next WRF release it think they are fixing this issue and recognizing that unidata is choosing to separate the C version and dependent Fortran versions of netCDF when built. See https://github.com/wrf-model/WRF/pull/1269.

Reply:
that is ok. I've added a few more changes to the run portion of the script including adding a new run script for the coupled model with shortwave feedback turned off.